supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc

```diff
@@ -323,13 +323,13 @@ There is no functional difference between the first server that was installed an
 *See also*:
 
 [[ingress]]
-==== image:images/yes.png[yes] Ingress (noun)
-*Description*: In Red Hat OpenShift, _Ingress_  is an API object developers can use to expose services through an HTTP(S) aware load balancing and proxy layer through a public DNS entry. The Ingress resource might further specify TLS options and a certificate, or specify a public CNAME that the OpenShift Ingress Controller should also accept for HTTP and HTTPS traffic. An administrator typically configures their Ingress Controller to be visible outside the cluster firewall and might also add additional security, caching, or traffic controls on the service content.
+==== image:images/yes.png[yes] ingress (noun) (adjective)
+*Description*: In Red Hat OpenShift, _Ingress_  is an API object that developers can use to expose services within the cluster through an HTTP(S) aware load balancer and a proxy layer via a public DNS entry. The `Ingress` resource defines the cluster-wide configuration for ingress traffic, and provides the ability to specify TLS options, a certificate, or a public CNAME that the OpenShift Ingress Controller (`IngressController`) can accept for HTTP(S) traffic. The OpenShift Ingress Operator uses the domain from the cluster `Ingress` object configuration as the domain for the default `IngressController`. An administrator typically configures the  `IngressController` to be visible outside the cluster firewall and might also add additional security, caching, or traffic controls on the service content. Furthermore, ingress also describes the incoming direction of network traffic. In Red Hat OpenShift, this traffic is described as entering (ingress) or leaving (egress) an OCP cluster.
```

Comment:
```suggestion
*Description*: In Red Hat OpenShift, _Ingress_  is an API object that developers can use to expose services within the cluster through an HTTP(S) aware load balancer and a proxy layer using a public DNS entry. The `Ingress` resource defines the cluster-wide configuration for ingress traffic, and provides the ability to specify TLS options, a certificate, or a public CNAME that the OpenShift Ingress Controller (`IngressController`) can accept for HTTP(S) traffic. Furthermore, _ingress_ also describes the incoming direction of network traffic. In Red Hat OpenShift, this traffic is described as entering (ingress) or leaving (egress) an OCP cluster.
```
The description is very long and detailed - not sure it needs to be so detailed.